orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/providers/gce/GoogleServerGroupCreator.groovy

```diff
@@ -59,6 +58,27 @@ class GoogleServerGroupCreator implements ServerGroupCreator, DeploymentDetailsA
     return [[(ServerGroupCreator.OPERATION): operation]]
   }
 
+  private String getImage(Stage stage) {
+    def stageContext = stage.getContext()
+    String image
+
+    if (stageContext.containsKey("imageSource") && stageContext.get("imageSource") == "artifact") {
```

Comment:
I think the `groovy` way to do this is `if (stageContext.imageSource == "artifact") {`